### DISCUSSION TITLE
My build is stuck at queued status it's not started even I wait 15-20 minutes

### DISCUSSION THREAD

**Discussion by MilanRibadiya** [codemagic-ci-cd/codemagic-docs#2448] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

-

### Expected results

I want to start my build 

### Actual results

Build is stuck at queued status

### Build id (optional)

6605c0d613fee65a6f9a7b98

_No response_

**Comment by DevTekoa** [comment]
Same problem here.

**Comment by mikhail-tokarev** [comment, marked as answer]
hey! please check status.codemagic.io page and this thread for more information https://github.com/orgs/codemagic-ci-cd/discussions/2424

**Reply by MilanRibadiya** [reply]
OK Thanks for answer...
